test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/ContractAutoExpirySpecs.java

```diff
@@ -606,6 +608,77 @@ private HapiSpec renewsUsingContractFundsIfNoAutoRenewAccount() {
+                                        ByteString.copyFromUtf8("My lovely NFT 2"))),
+                        createLargeFile(GENESIS, initcode, literalInitcodeFor("InstantStorageHog")),
+                        enableContractAutoRenewWith(minimalLifetime, 0),
+                        contractCreate(contractToRemove, new BigInteger("63"))
+                                .gas(2_000_000)
+                                .entityMemo("")
+                                .bytecode(initcode)
+                                .balance(0)
+                                .autoRenewSecs(minimalLifetime),
+                        tokenAssociate(contractToRemove, List.of(aFungibleToken, nonFungibleToken)),
+                        cryptoTransfer(
+                                moving(aFungibleAmount, aFungibleToken)
+                                        .between(TOKEN_TREASURY, contractToRemove),
+                                movingUnique(nonFungibleToken, 1L, 2L)
+                                        .between(TOKEN_TREASURY, contractToRemove)),
+                        getAccountBalance(contractToRemove)
+                                .hasTokenBalance(aFungibleToken, aFungibleAmount)
+                                .hasTokenBalance(nonFungibleToken, 2),
+                        getAccountBalance(TOKEN_TREASURY).hasTokenBalance(aFungibleToken, 0),
+                        getAccountInfo(TOKEN_TREASURY).hasOwnedNfts(0),
+                        /* sleep past the expiration:
+                         * (minimalLifetimeMillis * 1 second) + 500 ms (500 ms for extra time)
+                         */
+                        sleepFor((minimalLifetime * 1_000L) + 500L))
+                .when(
+                        // run transactions so the contract can be deleted
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)),
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)),
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)),
+                        sleepFor(2_000L), // wait for the record stream file to close
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)))
+                .then(
+                        // check that the contract is gone
+                        getContractInfo(contractToRemove)
+                                .hasCostAnswerPrecheck(INVALID_CONTRACT_ID),
+                        // And the fungible units were returned to the treasury
+                        getAccountBalance(TOKEN_TREASURY)
+                                .hasTokenBalance(aFungibleToken, aFungibleAmount),
+                        // And the NFTs are now owned by the treasury
+                        getTokenNftInfo(nonFungibleToken, 1L).hasAccountID(TOKEN_TREASURY),
+                        getTokenNftInfo(nonFungibleToken, 2L).hasAccountID(TOKEN_TREASURY)
+                        // TODO: re-enable after expiry throttling is fixed
```

Comment:
TODO

Reply:
Issue fixed, `TODO` no longer needed. I removed the `TODO` comment